src/mWater_collate/mWater_pull_collate.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "mWater_pull_collate"
+author: "Sam Struthers"
+date: "`r Sys.Date()`"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+
+#grab site metadata
+sampling_sites <- read_csv("data/water_sampling_sites.csv")
```

Comment:
I had to change the file location here, but I don't know why... Generally I was struggling with file locations. I don't know if this is a problem with this PR or with my local instance. Will have to check this again once it is merged.

Reply:
I also had issues with the file locations... not sure why...